sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/SessionStorage.java

```diff
@@ -0,0 +1,207 @@
+package com.microsoft.appcenter.persistence;
```

Comment:
Can we create a new package named `session` instead?

Reply:
If I rename to SessionContext is either root or utils, I am not sure why we would need a new package for just this class.